.github/workflows/push.yaml

```diff
@@ -53,3 +53,43 @@ jobs:
         with:
           name: redskyctl_linux_amd64
           path: dist/redskyctl-linux-amd64.tar.gz
+      - name: KinD (Kubernetes in Docker) Action
+        uses: engineerd/setup-kind@v0.3.0
+      - name: commands
+        run: |
+          kubectl get nodes -o wide
+          kubectl get node kind-control-plane -o yaml
+          kubectl get pods -n kube-system
+      - name: redsky quickstart
+        run: |
+          echo "Install redskyctl"
```

Comment:
We should probably skip this and use the artifact we just built

Reply:
I think you can use:

```
- uses: actions/download-artifact@v2
  with:
    name: redskyctl_linux_amd64
    path: ...
```